### PR TITLE
MIG mgmtsystem_system to V. 10.0

### PR DESCRIPTION
**Pull request by eugen-don**
MIG mgmtsystem_system to V. 10.0

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

File: mgmtsystem/__manifest__.py [hunk 1]
```diff
@@ -1,31 +1,15 @@
...
 {
     "name": "Management System",
-    "version": "9.0.1.0.0",
+    "version": "10.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Management System",
-    'images': ['images/mgmtsystem.png', 'images/mgmtsystem-hover.png'],
+    "complexity": "normal",
```
Comment (dreispt) [inline mgmtsystem/__manifest__.py]: complexity key is deprecated, please remove

File: mgmtsystem/security/mgmtsystem_security.xml [hunk 2]
```diff
@@ -18,22 +18,22 @@
         <field name="name">User</field>
         <field name="comment">Module user, with general read and write.</field>
         <field name="category_id" ref="module_category_management_system"/>
-        <field name="implied_ids" eval="[(4,ref('mgmtsystem.group_mgmtsystem_viewer'))]"/>
+        <field name="implied_ids" eval="(4,ref('mgmtsystem.group_mgmtsystem_viewer'))"/>
```
Comment (dreispt) [inline mgmtsystem/security/mgmtsystem_security.xml]: Per documentation, the square brackets are required.

File: mgmtsystem/views/menus.xml [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
+      web_icon_hover="images/mgmtsystem-hover.png"
       />
 
     <menuitem
       id="menu_mgmtsystem_main"
       name="Management System"
       parent="menu_mgmtsystem_root"
       sequence="0"
-      groups="group_mgmtsystem_viewer"/>
+      groups="base.group_user"/>
```
Comment (dreispt) [inline mgmtsystem/views/menus.xml]: Why is this change? It unnecessarily forces access to everyone.

**Review by dreispt — COMMENTED** [review]
I see some functional changes I'm not sure about.

**Comment by max3903** [comment]
@dreispt Can we merge?

File: mgmtsystem/views/menus.xml [hunk 4]
```diff
@@ -34,4 +34,4 @@
         parent="mgmtsystem.menu_mgmtsystem_root"
         groups="group_mgmtsystem_manager"
         sequence="100"/>
-</odoo>
+</odoo>
```
Comment (grindtildeath) [inline mgmtsystem/views/menus.xml]: Missing EOL

File: mgmtsystem/views/mgmtsystem_system.xml [hunk 5]
```diff
@@ -1,12 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-  <record model="ir.ui.view" id="system_form">
+  <data>
```
Comment (grindtildeath) [inline mgmtsystem/views/mgmtsystem_system.xml]: Why add data ?

File: mgmtsystem/models/mgmtsystem_system.py [hunk 6]
```diff
@@ -1,36 +1,23 @@
...
-##############################################################################
-
-from openerp import models, fields, _
+# Copyright 2012 Savoir-faire Linux <http://www.savoirfairelinux.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+import logging
+_logger = logging.getLogger(__name__)
```
Comment (grindtildeath) [inline mgmtsystem/models/mgmtsystem_system.py]: Do we need this ?

File: mgmtsystem/views/mgmtsystem_system.xml [hunk 7]
```diff
@@ -35,8 +39,10 @@
     <menuitem
```
Comment (grindtildeath) [inline mgmtsystem/views/mgmtsystem_system.xml]: Why not move it to views/menus.xml ?

**Review by grindtildeath — COMMENTED** [review]
Just a few things, otherwise looks quite good imo.

**Comment by eugen-don** [comment]
I did a rebase on 10.0 to compress commit history... sorry for that... could you review once more?

**Comment by eugen-don** [comment]
runbot, travis, and cc passed, this can be merged

File: mgmtsystem/__manifest__.py [hunk 8]
```diff
@@ -39,5 +23,5 @@
         'views/res_config.xml'
     ],
     "demo": [],
-    'installable': False,
+    'installable': True,
 }
```
Comment (sbidoul) [inline mgmtsystem/__manifest__.py:25]: Hi, In the changelog it's written "the module does no depends anymore on document_page module", however in the manifest the dependency on document_page is present. Which one is correct?

Comment (sbidoul) [reply mgmtsystem/__manifest__.py:25]: @eugen-don @max3903 @dreispt related, I notice there is a dependency on `board` that generates warnings during installation, and does not seem to be used inside this `mgtsystem` module.

The warnings/errors I get are 

```
WARNING test-mgtsystem-v10c odoo.modules.registry: Models have no table: board.board.
INFO test-mgtsystem-v10c odoo.modules.registry: Recreate table of model board.board.
ERROR test-mgtsystem-v10c odoo.modules.registry: Model board.board has no table.
```

Comment (eugen-don) [reply mgmtsystem/__manifest__.py:25]: Hi @sbidoul 
thanks nice spot.

Comment (eugen-don) [reply mgmtsystem/__manifest__.py:25]: @sbidoul 
i just checked the 9.0 __openerp__.py
and it had still board and document_page in the dependencies even though readme tells its not depending on document_page anymore.

I testet the base module and the other that ive ported, they all install without board, but some have document_page as a dependency.

So its save to remove board and document_page from dependencies in the core module.

edit:
it seems @naousse removed the dependency in his port to v 9.0 and added it back in this commit later on: https://github.com/OCA/management-system/commit/dc051b660ff1c8412cbc6b3f1ce212fa06b33f6d#diff-6061d36dd5e773bfa61a36521d74ca81

Comment (max3903) [reply mgmtsystem/__manifest__.py:25]: document_page should be a dependency of mgmtsystem_manual, _procedure and _work_instruction

Comment (eugen-don) [reply mgmtsystem/__manifest__.py:25]: @max3903 
at this moment mgmtsystem_manual and document_page_work_instruction depend both on document_page_approval and document_page_procedure depends on document_page_work_instruction.
document_page_approval depends on document_page

**Pull request merged**